htdocs/install/mysql/tables/llx_categorie_project_task.key.sql

```diff
@@ -0,0 +1,25 @@
+-- ============================================================================
```

Comment:
Can you rename the file into
htdocs/install/mysql/tables/llx_categorie_project_task-project.key.sql
So the sql file should be executed only module project is enabled.